### PR TITLE
Work on Layout

### PR DESCRIPTION
**Pull request by jmunozz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,6 +60,9 @@ typings/
 # IntelliJ IDEA
 .idea
 
+# Visual Code Workspace settings
```
Comment (atoulous) [inline .gitignore:63]: 👍

File: src/client/components/cell/Cell.scss [hunk 2]
```diff
@@ -1,3 +1,19 @@
-.example {
+.grid .row .cell {
```
Comment (atoulous) [inline src/client/components/cell/Cell.scss:1]: class grid is and should be only declared in Grid.scss ☝️

File: src/client/containers/NextPiece/NextPiece.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import './NextPiece.scss';
+import Grid from '../../components/grid/Grid';
+import { initGrid, forEachBlockInPiece } from '../../helpers';
+
+const NextPiece = ({ piecesQueue }) => {
```
Comment (atoulous) [inline src/client/containers/NextPiece/NextPiece.js:8]: please set propTypes in a new component because we will do anyway

**Review by atoulous — APPROVED** [review]
Nice one ! Just 2 things to do for me:

- simplified understanding of class name in declared their just one time in the good file
- do props validation

(normally: do unit test for each new component but yes its annoying)

File: src/client/containers/NextPiece/NextPiece.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  const nextPiece = pieceQueue[0];
```
Comment (atoulous) [inline src/client/containers/NextPiece/NextPiece.js:9]: piecesQueue not pieceQueue

Comment (atoulous) [reply src/client/containers/NextPiece/NextPiece.js:9]: actually it doesnt working 🤷‍♂️

File: src/client/containers/tetris/Tetris.js [hunk 5]
```diff
@@ -1,25 +1,23 @@
...
   <div className="tetris">
-    <Paper style={style} zDepth={2} rounded={false}>
-      <Score />
-      <Divider style={{ marginBottom: '10px' }} />
-      <Game />
-    </Paper>
+    <div className="tetris-meta">
+      <div className="tetris-meta-container">
+        <Score />
+        <NextPiece />
```
Comment (atoulous) [inline src/client/containers/tetris/Tetris.js:16]: ok so we have to send socket event when the pieces queue is empty

**Review by thifranc — COMMENTED** [review]
test

**Pull request merged**